keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -843,7 +1053,9 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
-- TBD (complete for beta)
+- For upgrade and rollback, both version skew scenarios were successfully tested. So
+  if, during upgrade or rollback, one of the API servers does not support the new WebSockets
+  functionality, the client will successfully fallback to the legacy SPDY streaming.
```

Comment:
This makes sense to me.  Note that question here is specifically Was the upgrade->downgrade->upgrade path tested". Let's say something directly to that.  I think the answer is no, but that there is no persisted state so it's N/A here.

Reply:
Added a direct response to the "upgrade->downgrade->upgrade" scenario.